docs/page-controller.md

```diff
@@ -117,18 +117,21 @@ addFilter( 'woocommerce_admin_reports_list', 'my-namespace', ( reports ) => {
 Register the report page with the controller:
 
 ```php
-wc_admin_register_page(
-	array(
-		'id'     => 'woocommerce-analytics-example',
-		'title'  => __( 'Example', 'my-textdomain' ),
+function add_report_menu_item( $report_pages ) {
+	$report_pages[] = array(
+		'id'     => 'example-analytics-report',
+		'title'  => __( 'Example', 'woocommerce-admin' ),
```

Comment:
Here, should we leave `my-textdomain` instead of `woocommerce-admin`? We want extension developers to use their own text domain instead of `woocommerce-admin`'s, right?
```suggestion
		'title'  => __( 'Example', 'my-textdomain' ),
```